tmd/Topology/tests/test_topology_analysis.py

```diff
@@ -64,3 +65,133 @@ def test_distance_horizontal():
 def test_distance_horizontal_unnormed():
     nt.ok_(analysis.distance_horizontal_unnormed(sample_data_0, sample_data_0)==0.0)
     nt.ok_(analysis.distance_horizontal_unnormed(sample_data_0, sample_data_1, bins=4) == 1.0)
+
+
+def test_closest_ph():
+
+    random_values = np.random.random(4).tolist()
```

Comment:
Just write the values manually.
As a general case, the less code is being run for producing a test, the better.
A code should have almost no logic inside.

Reply:
ok